style/style.css

```diff
@@ -1,7 +1,7 @@
 
 
 tr:nth-child(even) {
-  background: #E3E3E3;
+  background: #e3e3e32e;
```

Comment:
This won't work in IE11 but… I guess that doesn't matter

Reply:
Oh wait